src/Symfony/Bundle/FrameworkBundle/Command/BuildDebugContainerTrait.php

```diff
@@ -32,13 +33,13 @@ trait BuildDebugContainerTrait
      *
      * @throws \LogicException
      */
-    protected function getContainerBuilder(): ContainerBuilder
+    protected function getContainerBuilder(KernelInterface $kernel = null): ContainerBuilder
```

Comment:
why this? Isn't the kernel the same?

Reply:
I cannot do `$this->getApplication()->getKernel();` as the trait will do if we dont provide a Kernel. This trait is defined for `Commands`. Im hacking this... =)